src/content/quickstart/kind/_index.md

```diff
@@ -61,23 +61,12 @@ To manually verify the deployment follow the steps below.
 kubectl --kubeconfig output/kubeconfigs/kind-config-cluster3 create deployment nginx --image=nginx
 kubectl --kubeconfig output/kubeconfigs/kind-config-cluster3 expose deployment nginx --port=80
 kubectl --kubeconfig output/kubeconfigs/kind-config-cluster3 -n default apply -f - <<EOF
-apiVersion: lighthouse.submariner.io/v2alpha1
-kind: ServiceExport
-metadata:
-  name: nginx
-EOF
-kubectl --kubeconfig output/kubeconfigs/kind-config-cluster2 -n default  run --generator=run-pod/v1 tmp-shell --rm -i --tty --image quay.io/submariner/nettest -- /bin/bash curl nginx.default.svc.supercluster.local:8080
+subctl export service --namespace default nginx
```

Comment:
Please add short description on the last command, as it creates a new k8s object, that people my encounter first time. For example, you can add:
`export service` creates a K8s `ServiceExport` resource for the service, you've just exported. In the example above, `nginx` is created in `default` namespace (not specifying namespace will use the current one, as defined in kubeconfig).

Reply:
Done, more details about this option is available in https://github.com/submariner-io/submariner-website/pull/207